src/lib/components/Games/HolidayGame.svelte

```diff
@@ -0,0 +1,131 @@
+<script lang="ts">
+	import { browser } from '$app/environment';
+	import { holidayLists } from '$lib/assets/holidayLists';
+	import Board from '$lib/components/Board.svelte';
+	import GameSummary from '$lib/components/GameSummary.svelte';
+	import Timer from '$lib/components/Timer.svelte';
+	import { gameProgress } from '$lib/store';
+	import { formWord, makeWordUsed, type LetterData } from '$lib/utils/letterDataUtils';
+	import { createLetterData } from '$lib/utils/letterDataUtils';
+	import { checkIfValidWord } from '$lib/utils/wordUtils';
+
+	let now = new Date();
+	const wordList = holidayLists.find((list) => {
+		return now.getMonth() == list.date.getMonth() && now.getDate() == list.date.getDate();
+	})?.words;
```

Comment:
I feel like there are a couple of function lurking in here. This logic for checking if days are equal is duplicated elsewhere, and would be good to just write it once and use everywhere
```
const daysAreEqual = (d1: Date, d2: Date) =>
  d1.getMonth() === d2.getMonth() && d1.getDate() === d2.getDate();

const findWordListForToday = (wordLists: HolidayData[]) => {
  const today = new Date();
  const wordListWithDataForToday = wordLists.find(
    (list) => daysAreEqual(today, list.date)
  );
  return holidayDataForToday?.words;
}
```